lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -79,22 +82,36 @@ def heredoc?(node)
           args.loc.expression.source[0, 2] == SHOVEL
         end
 
+        def literal?(node)
```

Comment:
I will see if `astrolabe` would like to merge it. If so, we can just use it from there.